tools/generator/sdk/packages.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package sdk
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+const apiDirSuffix = "api"
+
+// TODO -- use the functions from the sdk itself instead when there is one in sdk
+func GetPackages(root string) ([]string, error) {
```

Comment:
We have a function in `tools/internal/packages/track1` to list all the packages - we could remove this and use that instead

Reply:
fixed